.github/workflows/synchronize-jira-xray.yml

```diff
@@ -20,15 +20,22 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v3
         with:
-          fetch-depth: 0 
+          fetch-depth: 0
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+          run_install: false
 
-      - name: Set up Python
-        uses: actions/setup-python@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
         with:
-          python-version: 3.x
+          node-version: 18
 
       - name: Install Dependencies
-        run: pip install requests
+        run: |
+          pnpm install axios
+          pnpm install form-data
```

Comment:
Can you install a specific major version of axios and form-data to avoid breaking changes on new releases